master/buildbot/reporters/gerrit_verify_status.py

```diff
@@ -135,6 +136,8 @@ def createStatus(self,
         if duration is not None:
             payload['duration'] = duration
 
+        payload = {'verifications': {name: payload}}
```

Comment:
I looked at the history of verify status plugin for several month, and could not see a version with current code's payload.